src/main/scala/mirroring/services/databases/JdbcPartitionedService.scala

```diff
@@ -43,62 +43,43 @@ class JdbcPartitionedService(
     options
   }
 
+  private lazy val lowerBound: String = getBounds().first().getString(1)
```

Comment:
What if the table is empty and there is empty dataset returned from `getBounds()`?